prediction_market_agent/agents/coinflip_agent/coinflip_agent.py

```diff
@@ -0,0 +1,13 @@
+import random
+import typing as t
+
+from prediction_market_agent_tooling.deploy.agent import DeployableAgent
+from prediction_market_agent_tooling.markets.agent_market import AgentMarket
+
+
+class DeployableCoinFlipAgent(DeployableAgent):
+    def pick_markets(self, markets: t.Sequence[AgentMarket]) -> t.Sequence[AgentMarket]:
+        return random.sample(markets, 1)
+
+    def answer_binary_market(self, market: AgentMarket) -> bool | None:
+        return random.choice([True, False])
```

Comment:
Should we make it a convention to have:

1. prediction_market_agent/agents/<agent_name>/<agent_name>.py: core code
2. prediction_market_agent/agents/<agent_name>/deploy.py: subclassing DeployableAgent

?

I guess that would mean just renaming this file delploy.py

Reply:
I see known_outcome_agent already uses (2), so I changed it to that one. Didn't notice it before, sorry!